### DISCUSSION TITLE
Anyone running on AWS?

### DISCUSSION THREAD

**Discussion by dpmccabe** [slskd/slskd#901] [General]
I'm thinking of running this app (or Nicotine+) on AWS, but I'm a little concerned about what flags might get raised when AWS sees the network traffic. Has anyone been doing this successfully for a while? Although the risk is probably pretty low, the worst-case scenario (account closure) is potentially very bad.

If you're using AWS but running the relay or a manual proxy elsewhere, where are you hosting it?

**Comment by imvikysharma** [comment]
It's understandable to be concerned about potential issues when running applications on AWS, but rest assured that many people have been using Nicotine+ on AWS without any issues.

That being said, it's always a good idea to be cautious and take appropriate measures to protect your account. One option could be to use a manual proxy hosted elsewhere to handle the network traffic, as you mentioned. Another option could be to set up a VPC (Virtual Private Cloud) in AWS to isolate the application's network traffic and restrict access to only authorized users.

In last I am recommending 1 book on AWS that can be very helpful. Do have a look

I would like to recommend a book on AWS

https://www.amazon.com/dp/B0BVJKH8PY?maas=maas_adg_AA06FAB7196A55FE3DBBEE7958CED70A_afap_abs&ref_=aa_maas&tag=maas

**Reply by jpdillingham** [reply]
Great answer!

I'll add that the biggest risk is that you may receive a DMCA takedown request.  Here's an article I found that describes how AWS handles them: https://recordinglaw.com/dmca-takedown-aws/

**Comment by sturia** [comment]
One way to address it, if your home router supports it, is to setup a site-to-site vpn from your public cloud instance to your router and route instance (or only soulseek) traffic through it. this way the only thing the cloud provider sees going back and forth is a p2p stream of encrypted packets between the instance and your home IP address.
furthermore, if you're using AWS, one way to make music file storing more cost effective is to leverage tools like s3fs or goofys to mount an S3 bucket -way cheaper/GB than block storage- like it was a hard drive and instruct slskd to use that as storage location
